src/cgns_internals.c

```diff
@@ -7039,7 +7039,7 @@ int cgi_write_model(double parent_id, cgns_model *model)
     }
 
      /* xModel_t */
-    sprintf(label,"%s_t",model->name);
+    sprintf(label,"%.30s_t",model->name);
```

Comment:
Suggest snprintf(label, 33, model->name);

Reply:
I don't have a preference for snprintf over sprintf, so it's up to you. CGNS only uses snprintf once and it is in the tests.